### PR TITLE
Release Java tools v11.1

### PR DESCRIPTION
**Pull request by comius**
Changes:
```
8cc0e261a3 cushon Include <limits>
960b7a99af Googler Adds an api-android target to the sharding API.
7749cf7579 Googler Use getRunfilesPath when constructing runfiles input manifest maps.
1489f0f4ca Timothy Klim Support Scala3 .tasty files
d113d74541 Liam Miller-Cushon Update turbine
41b3ab66b7 larsrc Create TestUtils methods to generate unique temp dirs, deprecate the current ones that are decidedly non-unique.
```

Fixes https://github.com/bazelbuild/bazel/issues/12605, https://github.com/bazelbuild/java_tools/issues/41

### CODE REVIEW

**Review by aiuto — COMMENTED** [review]
Please take a look at https://github.com/bazelbuild/bazel/pull/12810
If we go straight to that, this could be much easier. You would only have to update the urls and sha in one file.

**Review by meteorcloudy — APPROVED** [review]
The PR looks good. It's better to refactor as Tony suggested, but feel free to merge if it's urgent.

**Comment by comius** [comment]
> Please take a look at #12810
> If we go straight to that, this could be much easier. You would only have to update the urls and sha in one file.

Sure it looks better after refactoring. This is not that critical, I can wait with this PR and rebase once @aiuto is done.

File: distdir_deps.bzl [hunk 1]
```diff
@@ -227,10 +227,10 @@ DIST_DEPS = {
     "remote_java_tools_linux": {
         "aliases": ["remote_java_tools_test_linux", "remote_java_tools_linux_for_testing"],
         "archive": "java_tools_linux-v11.0.zip",
```
Comment (benjaminp) [inline distdir_deps.bzl:229]: Do these lines need to be updated?

Comment (comius) [reply distdir_deps.bzl:229]: They sure do. Thanks!

Comment (meteorcloudy) [reply distdir_deps.bzl:229]: Ah.. sorry for missing these!

Comment (comius) [reply distdir_deps.bzl:229]: Fixes here https://github.com/bazelbuild/bazel/pull/12824

**Pull request closed without merging**